src/snmalloc/backend/backend.h

```diff
@@ -5,7 +5,8 @@ namespace snmalloc
 {
   /**
    * This class implements the standard backend for handling allocations.
-   * It abstracts page table management and address space management.
+   * It is parameterised by its page table management (Pagemap) and
```

Comment:
Page table management is a bit confusing as a term for userspace things, which don't see architectural page tables.

Reply:
```suggestion
   * It is parameterised by its Pagemap management and
```